test/runtime/Policies.go

```diff
@@ -665,32 +668,58 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 		connectivityTest(allRequests, helpers.App2, helpers.Httpd1, BeTrue)
 	})
 
+})
+
+var _ = Describe("RuntimeValidatedPolicyImportTests", func() {
+	var once sync.Once
+	var logger *logrus.Entry
+	var vm *helpers.SSHMeta
+
+	initialize := func() {
+		logger = log.WithFields(logrus.Fields{"test": "RunPolicies"})
```

Comment:
Here the name should be changed to `RunPolicyImportTests`

Reply:
Fixed.